src/vm/interoputil.cpp

```diff
@@ -3715,108 +3715,110 @@ BOOL IsMethodVisibleFromCom(MethodDesc *pMD)
     }
 }
 
-//---------------------------------------------------------------------------
-// This method determines if a type is visible from COM or not based on 
-// its visibility. This version of the method works with a type handle.
-// This version will ignore a type's generic attributes.
-//
-// This API should *never* be called directly!!!
-BOOL SpecialIsGenericTypeVisibleFromCom(TypeHandle hndType)
+namespace
 {
-    CONTRACTL
+    //---------------------------------------------------------------------------
+    // This method determines if a type is visible from COM or not based on
+    // its visibility. This version of the method works with a type handle.
+    // This version will ignore a type's generic attributes.
+    //
+    // This API should *never* be called directly!!!
+    BOOL SpecialIsGenericTypeVisibleFromCom(TypeHandle hndType)
```

Comment:
Just mark it `static` to achieve the same effect with less ceremony?

Reply:
Yeah... adding `static` does that. I prefer this approach, but in the spirit of consistency I will change it to `static`.